Index: app/src/main/java/com/example/first_app/Login.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/first_app/Login.java	(revision 4a74e9e112b1c1fb2bfb2a2aae157348c607810c)
+++ app/src/main/java/com/example/first_app/Login.java	(date 1599756386031)
@@ -13,7 +13,7 @@
 
 
 public class Login extends AppCompatActivity {
-    Button callSignUp;
+    Button singup_btn;
 
 
     @Override
@@ -25,7 +25,7 @@
         setContentView(R.layout.activity_login);
 
 
-        callSignUp.setOnClickListener(new View.OnClickListener() {
+        singup_btn.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
                 Intent intent = new Intent(Login.this,SingUp.class);
Index: app/src/main/java/com/example/first_app/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/first_app/MainActivity.java	(revision 4a74e9e112b1c1fb2bfb2a2aae157348c607810c)
+++ app/src/main/java/com/example/first_app/MainActivity.java	(date 1599756712653)
@@ -43,7 +43,7 @@
         new Handler().postDelayed(new Runnable() {
             @Override
             public void run() {
-                Intent intent = new Intent(MainActivity.this, Login.class);
+                Intent intent = new Intent(MainActivity.this, SingUp.class);
 
                 Pair[] pairs = new Pair[2];
                 pairs[0] = new Pair<View, String>(image, "logo_image");
